README

```diff
@@ -241,7 +241,7 @@ REQUIREMENTS:
           DESTDIR=... meson install -C build/
 
         A tarball can be created with:
-          git archive --format=tar --prefix=systemd-222/ v222 | xz > systemd-222.tar.xz
+          v=222 && git archive --prefix=systemd-$v/ v$v | zstd >systemd-$v.tar.zstd
```

Comment:
222 -> 250?? This is not necessary to follow the recent release, but 222 is too old...